WalletWasabi.Fluent/ViewModels/CoinJoinProfiles/CoinJoinProfileViewModelBase.cs

```diff
@@ -0,0 +1,19 @@
+using Avalonia.Media;
+
+namespace WalletWasabi.Fluent.ViewModels.CoinJoinProfiles;
+
+public abstract class CoinJoinProfileViewModelBase : ViewModelBase
+{
+	public abstract string Title { get; }
+
+	public abstract string Description { get; }
+
+	public abstract IImage Icon { get; }
+
+	public virtual bool AutoCoinjoin { get; } = true;
+	public virtual int MinAnonScoreTarget { get; } = 5;
+
+	public virtual int MaxAnonScoreTarget { get; } = 10;
```

Comment:
I think the profile should be consistent even if the default is changed. However, practically it does not matter for new users because they have to go through this page before they can have a new wallet. Existing users will get the default.